src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private int firstLeg;
+    private int secondLeg;
+
+    public RightTriangle(String color, int firstLeg, int secondLeg, String name) {

```

Comment:
Avoid passing expressions directly into the constructor. Instead, declare separate variables for the `firstLeg` and `secondLeg` parameters and pass those variables to the constructor for better readability.